psydac/api/ast/nodes.py

```diff
@@ -755,18 +760,16 @@ def __init__(self, weights=True):
     @property
     def weights( self ):
         return self._weights
-#==============================================================================
-class PlusLocalTensorQuadrature(LocalTensorQuadrature):
-    pass
 
 #==============================================================================
 class PlusGlobalTensorQuadrature(GlobalTensorQuadrature):
-    pass
+    """This class represents the quadrature points and weights in the plus boundary of an interface.
+    """
 
 #==============================================================================
 class TensorQuadrature(ScalarNode):
-    """
-    """
+    """This class represents the quadrature point and weight in a domain."""
```

Comment:
Do you meant "quadrature points and weights" (plural)?

Reply:
here is singular, because the quadrature point and the weight are fixed

Reply:
And that's because it inherits from `ScalarNode`, right?

Reply:
yes

Reply:
I did not pay attention to this. I was just assuming that `GlobalTensorQuadrature` was a subclass of `TensorQuadrature` as the name suggest. So I think that we need better names for these classes, and the minimal change I can imagine is appending `Grid` to the array classes:

- `GlobalTensorQuadrature` -> `GlobalTensorQuadratureGrid`
- `LocalTensorQuadrature` -> `LocalTensorQuadratureGrid`

We can also use `Array` instead of `Grid`, if you think that's more appropriate

Reply:
okay, `GlobalTensorQuadratureGrid` sounds good

Reply:
done in [20f4c78](https://github.com/pyccel/psydac/pull/160/commits/20f4c788cb999572c4a156bf4eb893e1fc3bdc02)